includes/polling/os/aix.inc.php

```diff
@@ -0,0 +1,5 @@
+<?php
+$serial = snmp_get($device, 'aixSeSerialNumber.0', '-Oqv', 'IBM-AIX-MIB');
+$hardware = snmp_get($device, 'aixSeMachineType.0', '-Oqv', 'IBM-AIX-MIB');
+$aix_version = preg_split('/\s+/', $device['sysDescr']);
```

Comment:
Your version code isn't working for the previous data that worked using the code in unix.inc.php

Reply:
unix.inc.php inc wrongly identifies serial and hardware. includes/polling/os/aix.inc.php corrects isuues.
i can correct unix.inc.php in separate pull request. 
Let's get rid of includes/polling/os/aix.inc.php for now. Storage is more important.

Reply:
Sounds great.  I am always in favor of small focus'd PRs :)